app/locale/de_DE/Mage_Customer.csv

```diff
@@ -28,7 +28,9 @@
 "Admin","Admin"
 "All","Alle"
 "All Store Views","Alle StoreViews"
-"All countries","Alle Länder"
+"All countries","Allef you are a registered VAT Länder"
```

Comment:
you already deleted your repo? Then I would say, fork again, fix again, PR again :-/